website/docs/r/compute_project_metadata.html.markdown

```diff
@@ -8,7 +8,10 @@ description: |-
 
 # google\_compute\_project\_metadata
 
-Manages metadata common to all instances for a project in GCE.
+Manages metadata common to all instances for a project in GCE. If you
+want to manage only single key/value pairs within the project metadata
+rather than the entire map, then use
```

Comment:
Hmm, good point. I see in the documentation about it being a set of metadata field docs, but nowhere about it being a map. Will update.